include/translate-option.php

```diff
@@ -216,8 +229,10 @@ protected function register_string_recursive( $context, $option, $values, $key )
 					$this->register_string_recursive( $context, $n, $value, $key );
 				}
 			}
-		} else {
-			PLL_Admin_Strings::register_string( $option, $values, $context, true );
+		} elseif ( is_scalar( $values ) ) {
+			$string         = (string) $values;
+			$this->hashes[] = md5( "$string|$option|$context" );
```

Comment:
I would have a dedicated method to get the hash. I see two benefits:
- Enhanced readability
- Easier maintenance (even though it's already encapsulated in *one* class)

Just a suggestion.